cmd/skywire-cli/commands/config/gen.go

```diff
@@ -354,6 +358,13 @@ var genConfigCmd = &cobra.Command{
 		if ver != "" {
 			conf.Common.Version = ver
 		}
+		if conf.Transport.PublicAutoconnect == publicautoconn {
+			conf.Transport.PublicAutoconnect = !publicautoconn
```

Comment:
Why are we using the negation of the value set? Seems unituitive. Also, why are we comparing values here with the default conf value and not just setting it in config if the flag was set? The same question applies to below code block.